src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Abstractions/ElementContainer.cs

```diff
@@ -9,17 +9,17 @@ namespace Microsoft.DotNet.ApiCompatibility.Abstractions
     /// Class to wrap an Element of T with it's <see cref="MetadataInformation"/>.
     /// </summary>
     /// <typeparam name="T">The type of the Element that is holded</typeparam>
-    public readonly struct ElementContainer<T> where T : ISymbol
+    public class ElementContainer<T> where T : ISymbol
```

Comment:
Moving to a struct as ElementContainer needs to be nullable. In the case of a struct, it would be wrapped into a Nullable<T> which isn't what we generally want. Also this shouldn't matter in any way as the numer of ElementContainers is limited (its equal to the number of assemblies passed in).